.github/workflows/ci-icefall.yml

```diff
@@ -0,0 +1,89 @@
+name: CI PyTorchIcefall
+on:
+  # on manual trigger
+  workflow_dispatch:
+
+  # on pull requests
+  pull_request:
+    paths-ignore:
+      - '*.md'
+
+  # on merge queue
+  merge_group:
+
+  # when pushing to main or dev branches
+  push:
+    branches:
+      - main
+      - dev*
+
+  # scheduled CI flow daily
+  schedule:
+    - cron: '0 8 * * 0'
+
+jobs:
+  test_icefall:
+    name: PyTorchIcefall
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y -q install ffmpeg libavcodec-extra
+          python -m pip install --upgrade pip setuptools wheel
+          pip3 install -r requirements_test.txt
+          apt-get update \
+          && apt-get install -y \
+              libgl1-mesa-glx \
+              libx11-xcb1 \
+              git \
+              gcc \
+              mono-mcs \
+              libavcodec-extra \
+              ffmpeg \
+              curl \
+              libsndfile-dev \
+              libsndfile1 \
```

Comment:
Is there a reason why this is done as two `apt install` steps? Can't they be combined into one?